src/backends/custom/custom.h

```diff
@@ -249,6 +287,15 @@ TRTIS_CUSTOM_EXPORT int CustomExecute(
     void* custom_context, uint32_t payload_cnt, CustomPayload* payloads,
     CustomGetNextInputFn_t input_fn, CustomGetOutputFn_t output_fn);
 
+/// See CustomExecute.
+TRTIS_CUSTOM_EXPORT int CustomExecuteVer2(
+    void* custom_context, uint32_t payload_cnt, CustomPayload* payloads,
+    CustomGetNextInputVer2Fn_t input_fn, CustomGetOutputVer2Fn_t output_fn);
+
+/// Return the custom version. For custom backend that doesn't define this
```

Comment:
Move this above CustomInitialize. Here is where we can define what the versions are.

"Get the custom version. For a custom backend that doesn't define this entry point, the inference server will assume the backend version is 1. The currently supported versions are defined below, returning any other version is an error:

Version 1: Input and output tensors must be communicated via system memory (i.e. CPU memory). The CustomExecute function must be defined and CustomGetNextInputFn_t and CustomGetOutputFn_t define the function signature for the input and output callbacks.

Version 2: Input and output tensors may be communicated by both system memory and GPU memory. The CustomExecuteV2 function must be defined and CustomGetNextInputV2Fn_t and CustomGetOutputV2Fn_t define the function signature for the input and output callbacks.

\return the custom version.